src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working.");
```

Comment:
The message 'Bulldozer started working.' could be more informative. According to the checklist, it should indicate the type of machine that is working. Consider including the class name in the message.